javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,85 @@
+let yourName = "";
+const toDosList = [];
+function getReply(command) {
+
+    if (command === "Hello my name is Benjamin" && !yourName) {
+        yourName += "Benjamin";
+        return "Nice to meet you " + yourName;
+    } else if (command === "Hello my name is Benjamin" && yourName) {
+        return "Yeah I know.. Nice to see you again by the way";
+    }
```

Comment:
Can you change this code handle a different name? I want to say "Hello my name is Cameron", or use another name.

Reply:
Done with this.. Should I merge it now as it is?

Reply:
I made one more suggestion about the name Benjamin. Once that is done, you can merge it.